src/utils/components/PodSelectorModal/selector-input.jsx

```diff
@@ -0,0 +1,200 @@
+import * as React from 'react';
+import * as TagsInput from 'react-tagsinput';
+import * as classNames from 'classnames';
+import * as _ from 'lodash';
+
+import { Label as PfLabel } from '@patternfly/react-core';
+
+import { selectorToString } from '../Selector/utilts';
+
+import * as k8sSelectorRequirement from './selector-requirement';
```

Comment:
Why not import the function by name since there's only one function in that file?